src/integrationTest/java/uk/gov/hmcts/reform/judicialbooking/config/SwaggerPublisher.java

```diff
@@ -20,19 +21,24 @@
  * Built-in feature which saves service's swagger specs in temporary directory.
  * Each travis run on master should automatically save and upload (if updated) documentation.
```

Comment:
Can we update to refere to workflow rather than travis: `.github/workflows/swagger.yml`

Reply:
comments updated

Reply:
Resolved in 4092eb927f1291f8b8d05ebc8f284efcf2def527